src/sentry/static/sentry/app/views/stream/organizationSavedSearchSelector.jsx

```diff
@@ -77,21 +105,46 @@ const Container = styled.div`
   }
 `;
 
+const SearchTitle = styled.strong`
+  display: block;
+  max-width: 100%;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  color: ${p => p.theme.gray5};
+  padding: 0;
+  background: inherit;
+`;
+
+const SearchQuery = styled.code`
+  display: block;
+  max-width: 100%;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+  color: ${p => p.theme.gray5};
+  padding: 0;
+  background: inherit;
+`;
+
 const StyledMenuItem = styled(MenuItem)`
+  position: relative;
+
   & a {
     /* override shared-components.less */
     padding: ${space(0.25)} ${space(1)} !important;
   }
-  & span,
-  & code {
+
+  & button {
```

Comment:
I tried to use the `${Button}` component instead of the html element selector but emotion barfed with `TypeError: "Cannot call a class as a function"`

Reply:
It's probably because Button is a react component and not a "styled component"

Reply:
That makes sense. Adding another styled component for this use-case seems unnecessary. With the design still influx I'd like to proceed with what I have as it is all going to change anyways.